src/averageAges.js

```diff
@@ -14,12 +14,17 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+function calculateMenAverageAge(people, century = 0) {
```

Comment:
```suggestion
function calculateMenAverageAge(people, century) {
```
no need to define it as a default value
undefined is a falsy value, so you may use it in ternary like this:
  `century ? //some code : //some code`
and it will work correctlyy